### DISCUSSION TITLE
Is there a website for asreview?

### DISCUSSION THREAD

**Discussion by bagheria** [asreview/asreview#525] [Q&A] [answered]
Is there a website available for asreview which introduces the project in detail?

**Comment by J535D165** [comment, marked as answer]
To get an overview of the ASReview project, see:

- ASReview project website http://asreview.nl/ 
- ASReview documentation https://asreview.readthedocs.io/ (especially [The Case of Systematic Reviewing](https://asreview.readthedocs.io/en/latest/intro/about.html#the-case-of-systematic-reviewing)) 
- Scientific publication about the project https://doi.org/10.1038/s42256-020-00287-7
